templates/common/upgrade/merge.go

```diff
@@ -251,11 +251,15 @@ func mergeAll(ctx context.Context, p *commitParams, dryRun bool) error {
 }
 
 const (
+	// These are appended to files that need manual merge conflict resolution.
 	suffixLocallyAdded                  = ".abcmerge_locally_added"
 	suffixLocallyEdited                 = ".abcmerge_locally_edited"
 	suffixFromNewTemplate               = ".abcmerge_from_new_template"
 	suffixFromNewTemplateLocallyDeleted = ".abcmerge_locally_deleted_vs_new_template_version"
 	suffixWantToDelete                  = ".abcmerge_template_wants_to_delete"
+
+	// This is the beginning of all the above suffixes.
+	conflictSuffixBegins = ".abcmerge_"
```

Comment:
nits: use such suffix for the above variables like `suffixLocallyAdded`, `suffixLocallyEdited` and so on?

Reply:
Hm I think I want to keep the name different, because that makes it more obvious that this isn't just another suffix.